membership-token/js/test/actions/close-market.ts

```diff
@@ -0,0 +1,82 @@
+import test from 'tape';
+import {
+  assertConfirmedTransaction,
+  assertError,
+  defaultSendOptions,
+  PayerTransactionHandler,
+} from '@metaplex-foundation/amman';
+
+import { Connection, Keypair, SYSVAR_CLOCK_PUBKEY, Transaction } from '@solana/web3.js';
+
+import { createAndSignTransaction, logDebug } from '../utils';
+
+import { createCloseMarketInstruction } from '../../src/instructions';
+
+type CloseMarketParams = {
+  test: test.Test;
+  transactionHandler: PayerTransactionHandler;
+  payer: Keypair;
+  connection: Connection;
+  market: Keypair;
+};
+
+export const closeMarket = async ({
+  test,
+  transactionHandler,
+  payer,
+  connection,
+  market,
+}: CloseMarketParams): Promise<void> => {
+  const instruction = await createCloseMarketInstruction({
+    market: market.publicKey,
+    owner: payer.publicKey,
+    clock: SYSVAR_CLOCK_PUBKEY,
+  });
+
+  const marketTx: Transaction = await createAndSignTransaction(
```

Comment:
Let's make this method to return transaction and remove sending of it from here. That way we can remove closeMarketLimitedDuration